Source/EmbeddedBoundary/WarpXFaceExtensions.cpp

```diff
@@ -11,10 +11,135 @@
 #include <AMReX_MultiFab.H>
 
 
+/**
+* \brief Get the value of arr in the neighbor (i_n, j_n) on the plane with normal 'dim'.
+*        I.E. If dim==0 it return arr(i, j + i_n, k + j_n),
+*             if dim==1 it return arr(i + i_n, j, k + j_n),
+*             if dim==2 it return arr(i + i_n, j + j_n, k)
+*
+* \param[in] arr \c Array4 To be accessed
```

Comment:
The type will be shown anyway with the name, so we can spare the refeference
```suggestion
* \param[in] arr the data to be read
```